.github/workflows/install_snippet.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -o errexit -o nounset -o pipefail
+
+# Set by GH actions, see
+# https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables
+TAG=${GITHUB_REF_NAME}
+
+
+cat << EOF
+## Intsall
+
+See install instructions in [README.md](https://github.com/aspect-build/aspect-cli/blob/${TAG}/README.md).
```

Comment:
Perhaps give the quick MacOS brew instructions as well? `brew install aspect-build/aspect/aspect` and move the `See install instructions ...` to the bottom?

A link to https://www.aspect.build/cli would be good as well.